hw/ip/gpio/data/gpio.hjson

```diff
@@ -110,7 +110,9 @@
       act:     "rcv",
       package: "",
       desc:    '''
-               This signal is pulsed high by the power manager after reset in order to sample the HW straps.
+               The strap enable signal is used to make the snapshot of the gpio_in.
+               This only has an effect the first time it goes high.
+               The following strap_en driven after that, it will not be taken into consideration.
```

Comment:
I think this text is a bit wonky. Can't you just say "The behaviour of this signal after that event will have no effect" ?